app/uk/gov/hmrc/teamsandrepositories/controller/WebhookController.scala

```diff
@@ -43,6 +48,26 @@ class WebhookController @Inject()(
       }
     }
 
+    def deletedRepositoryEvent(repoName: String): Future[Unit] = {
+      repositoriesPersistence.findRepo(repoName).flatMap {
+        case Some(repo) =>
+          for {
+            set <- deletedRepositoriesPersistence.set(Seq(DeletedGitRepository.fromGitRepository(repo, Instant.now())))
+            _   <- persistingService.repositoryDeleted(repoName)
+          } yield {
+            set match {
+              case true => logger.info(s"Deleted repository event: $repoName successfully set")
+              case false => logger.info(s"Deleted repository event: $repoName unsuccessfully set")
```

Comment:
The `set` returned doesn't actually say if it was successful or not - it's returning `wasAcknowledged` which isn't of much use to the client. Given the default mongo configuration, unacknowledged updates would lead to failed futures. I think it would be better to for `set()` to just return `Future[Unit]` and we don't need this log (or just change to `Deleted repository event processed`).
Same below.